bool autoActive
float32 goto_lat
float32 goto_lon